docs/upgrade-notes/data-migrations-rds.md

```diff
@@ -0,0 +1,93 @@
+---
+id: data_migrations_rds
+title: Running data migrations for RDS
+hide_title: false
+---
+
+## Background
+
+This release of Cumulus (x.x.x) integrates with RDS and creates a new PostgreSQL database for archiving Cumulus data (e.g. granules, files, executions).
```

Comment:
I'm pretty sure we're 100% on it being v10.0.0

Reply:
Are we? because the CHANGELOG in master doesn't show any breaking changes after v8.0.0:

https://github.com/nasa/cumulus/blob/master/CHANGELOG.md

So why would the pre-RDS release be 9.0.0?

Reply:
Hmm, I thought that at least one of the outstanding PRs to go in had a breaking change.    I may be mistaken.   In either case, flagging for update.